modules/ROOT/pages/endpoint-as2-receive.adoc

```diff
@@ -13,105 +13,33 @@ Required: Yes +
 |Describe any important details for this endpoint. +
 Required: No +
 
-| AS2 URL (Automatically generated at time of deployment)
+| AS2 URL
 | The HTTP or HTTPS address where you receive AS2 messages.
-The port is automatically generated when you deploy a message flow that uses this endpoint.
-You can see it by selecting the receiving endpoint from your deployed message flow.
-The hostname displayed is created by your administrator at setup.
 
+When deploying to CloudHub with a public Shared Load Balancer, the URL is automatically generated when you deploy a message flow that uses this endpoint.
```

Comment:
unclear reference: the URL is not deploying....
```suggestion
The URL is automatically generated when you use a shared load balancer to deploy a message flow to CloudHub.
```

Reply:
This text has been removed as it is no longer accurate.

Reply:
We do not want to bring the topic of load balancer at all. 
We just need to have the below with respect to CloudHub.

`When deploying to CloudHub the URL is automatically generated when you deploy a message flow that uses this endpoint. You can see it by selecting the receiving endpoint from your deployed message flow.`